package.json

```diff
@@ -1,39 +1,69 @@
 {
-  "name": "deprecation-crawler",
-  "version": "0.0.0-development",
-  "description": "",
-  "main": "index.js",
-  "bin": "cli.js",
+  "name": "deprecation-manager",
+  "version": "0.0.0",
+  "license": "MIT",
   "scripts": {
-    "start": "npm run build-and-pack-and-run-clean",
-    "crawl": "npx ./dist/index.js",
-    "start-clean": "npm run cleanup && npm start",
-    "prebuild": "rimraf ./dist",
-    "build": "tsc",
-    "postbuild": "cpy ./package.json ./dist && cpy ./cli.js ./dist",
-    "local": "ts-node lib/local.ts",
-    "build-and-pack": "npm run build && cd dist && npm pack && cd ..",
-    "cleanup": "rimraf ./deprecations ./deprecation-crawler.config.json && git checkout -- ./test-folder-for-crawling/**",
-    "build-and-pack-and-run": "npm run build-and-pack && npm run crawl",
```

Comment:
Where did the cleanup script go? It was quite helpful for resetting in development. :)

Reply:
I removed it because I think we didn't need it anymore.
How I see it, this was needed to run the crawler against the `test-folder-for-crawling` folder and to reset it. This is been replaced with the tests.

If my thinking it wrong, or if find it's still valuable we can add it back.

Reply:
Got it!! So i will experiment with tests. :)